src/codegen.cpp

```diff
@@ -519,14 +518,12 @@ typedef struct {
     bool vaStack;      // varargs stack-allocated
     bool sret;
     int nReqArgs;
-    std::vector<bool> boundsCheck;
-    std::vector<bool> inbounds;
 
     CallInst *ptlsStates;
     Value *signalPage;
 
-    llvm::DIBuilder *dbuilder;
     bool debug_enabled;
+    bool is_inbounds{false};
```

Comment:
Hm, I must've never tried this, or done it wrong? Looks like that works, thanks!